src/main/java/duke/misc/Ui.java

```diff
@@ -0,0 +1,17 @@
+package duke.misc;
+
+/**
+ * Ui class to encapsulate display messages and handle input/output of duke.
+ */
+public class Ui {
+    /** Welcome message. */
+    public static final String WELCOME_MSG = "Hi I am Annie!\nHow can I assist you?";
+
+    static final String GOODBYE_MSG = "Bye. See you soon!";
```

Comment:
I like how the messages are so concisely organised into a class. Perhaps explicitly adding the access modifier will make the code perfect